src/main/java/org/folio/linked/data/service/impl/BatchIndexServiceImpl.java

```diff
@@ -23,28 +31,33 @@
 @RequiredArgsConstructor
 public class BatchIndexServiceImpl implements BatchIndexService {
 
-  private final KafkaSearchSender kafkaSearchSender;
   private final EntityManager entityManager;
+  private final KafkaSearchMessageMapper<LinkedDataWork> searchBibliographicMessageMapper;
+  @Qualifier("bibliographicIndexEventProducer")
+  private final FolioMessageProducer<ResourceIndexEvent> bibliographicIndexEventProducer;
```

Comment:
We have the same 2 fields doing the same job in WorkSearchCreateEventProducer.
Could we do a last step of refactoring to get rid of the copy-paste in this class as well ? :)
I suspect we have to only make CreateResourceEventProducer#produce method return boolean as a flag of successful/not message sending